packages/peregrine/lib/talons/CommunicationsPage/communicationsPage.gql.js

```diff
@@ -21,10 +21,6 @@ export const GET_CUSTOMER_SUBSCRIPTION = gql`
 `;
 
 export default {
-    mutations: {
-        setNewsletterSubscriptionMutation: SET_NEWSLETTER_SUBSCRIPTION
-    },
-    queries: {
-        getCustomerSubscriptionQuery: GET_CUSTOMER_SUBSCRIPTION
-    }
+    getCustomerSubscriptionQuery: GET_CUSTOMER_SUBSCRIPTION,
+    setNewsletterSubscriptionMutation: SET_NEWSLETTER_SUBSCRIPTION
```

Comment:
I kind of like this better - we're already naming things `fooQuery` and `barMutation` anyways. No need to put another layer around them to indicate the type.

Reply:
@supernova-at commented about this recently, and I fully agree, it became redundant since we wanted to preserve the non-appended version for executing mutations (or lazy queries), eg. `setNewsletterSubscription` will execute the mutation.

Reply:
DevTime Discussion: Flatten out to `operations` and enforce operations append what type they are to the end of their name.